code/modules/chemistry/Reagents-FoodDrink.dm

```diff
@@ -69,7 +69,7 @@ datum
 				return
 
 			reaction_mob(var/mob/M, var/method=TOUCH, var/volume, var/paramslist = 0)
-				..()
+				. = ..()
 				if(M.mob_flags & IS_BONEY)
 					M.HealDamage("All", clamp(1 * volume, 0, 20), clamp(1 * volume, 0, 20)) //put a cap on instant healing
```

Comment:
should set `. = FALSE` here though, since it's being used for a touch reaction at this point

Reply:
you can splash milk on skeletons to heal them??? wild

Reply:
I did not know this until now either

Reply:
looks like it was changed in #9977. hm

Reply:
Amended

Would note the following:
- Healing triggers on touch or ingest, unless the reagent cap is full in which case touch only. The cap is reached very rapidly with critters so they need to transfer milk to a beaker to use it properly. Skeleton race gets the benefit of both.
- Double dipping with bleed reagent transfer to heal yourself is a thing but seemed to be net negative when testing.